AGENTS.md

```
@@ -0,0 +1,13 @@
+# AI Agent Instructions
+
+## Git Commits
+
+When creating git commits with AI assistance, always add a `Co-authored-by` trailer identifying both the AI agent and model:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)